lib/tests/modules/type-deprecation.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, ... }: {
+
+  options.simple = lib.mkOption {
+    type = lib.mkOptionType {
+      name = "simple";
+      deprecationMessage = "simple shall not be used!";
```

Comment:
I prefer kind messages (doesn't read like yelling) even in tests, `simple shall not be used.`

Reply:
Hehe fair enough, changed that :)